src/pages/staticPages/tools/oncoprinter/Oncoprinter.tsx

```diff
@@ -229,6 +235,11 @@ export default class Oncoprinter extends React.Component<IOncoprinterProps, {}>
     private oncoprintRef(oncoprint:OncoprintJS) {
         this.oncoprint = oncoprint;
 
+        onMobxPromise(this.props.store.alteredSampleIds,
```

Comment:
seems like a kinda weird place to put this no?

Reply:
sure that makes sense